database/migrations/2021_12_09_172010_create_contact_info_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateContactInfoTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement("
+            CREATE TABLE contact_info (
+                `customer_id` BIGINT UNSIGNED NOT NULL,
+                `phone_number` VARCHAR(20) NOT NULL,
+                `email` VARCHAR(50) NOT NULL,
+                PRIMARY KEY (`customer_id`),
+                UNIQUE KEY (`customer_id`),
```

Comment:
You don't need unique key if you already have Primary key for it.

Reply:
Deleted unique key